deb/brave-keyring/DEBIAN/control

```diff
@@ -1,7 +1,6 @@
 Package: brave-keyring
 Architecture: all
-Maintainer: Ben Kero <bkero@brave.com>
+Maintainer: Brave DevOps <devops@brave.com>
```

Comment:
I'd suggest using the same email address as the `brave-browser` package's changelog and our GPG key (as well as the Google Play entry for the Brave for Android app):

    Brave Software <support@brave.com>

That way we're not drawing attention to the devops email address.